doc/tutorial/statistical_inference/unsupervised_learning.rst

```diff
@@ -41,61 +41,37 @@ algorithms. The simplest clustering algorithm is :ref:`k_means`.
     >>> print(y_iris[::10])
     [0 0 0 0 0 1 1 1 1 1 2 2 2 2 2]
 
-.. |k_means_iris_bad_init| image:: /auto_examples/cluster/images/sphx_glr_plot_cluster_iris_003.png
-   :target: ../../auto_examples/cluster/plot_cluster_iris.html
-   :scale: 63
-
-.. |k_means_iris_8| image:: /auto_examples/cluster/images/sphx_glr_plot_cluster_iris_001.png
-   :target: ../../auto_examples/cluster/plot_cluster_iris.html
-   :scale: 63
-
-.. |cluster_iris_truth| image:: /auto_examples/cluster/images/sphx_glr_plot_cluster_iris_004.png
-   :target: ../../auto_examples/cluster/plot_cluster_iris.html
-   :scale: 63
-
 .. warning::
 
     There is absolutely no guarantee of recovering a ground truth. First,
     choosing the right number of clusters is hard. Second, the algorithm
     is sensitive to initialization, and can fall into local minima,
     although scikit-learn employs several tricks to mitigate this issue.
 
-    .. list-table::
-        :class: centered
-
-        *
-
-            - |k_means_iris_bad_init|
+    |
 
-            - |k_means_iris_8|
+    .. figure:: /auto_examples/cluster/images/sphx_glr_plot_cluster_iris_003.png
+       :target: ../../auto_examples/cluster/plot_cluster_iris.html
+       :scale: 63
 
-            - |cluster_iris_truth|
+       **Bad initialization**
```

Comment:
The current version seems to render just fine? (even better since images are centered)

Reply:
I have removed all the tables because images are not readable on small screens.
Image tables are not a good idea in general on small screens.